### PR TITLE
Document Amazon S3 source

### PR DESCRIPTION
**Pull request by antoineco**
[Render](https://github.com/antoineco/triggermesh-docs/blob/s3-src/docs/sources/awss3.md)

### CODE REVIEW

File: docs/sources/awss3.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+```
+arn:aws:s3:{awsRegion}:{awsAccountId}:{bucketName}
+```
+
+![S3 Bucket ARN](../images/awss3-source/arn-region-1.png)
+
+### API Credentials
+
+The TriggerMesh event source for Amazon S3 authenticates calls to the AWS API using an [Access Key][accesskey]. The page
+at this link contains instructions to create an access key when signed either as the root user or as an IAM user. Take
```
Comment (sebgoa) [inline docs/sources/awss3.md:30]: The page at which link ? is that referring to the previous link ?

Comment (antoineco) [reply docs/sources/awss3.md:30]: You're right that's some pretty poor phrasing here. I'll change this sentence.

Comment (antoineco) [reply docs/sources/awss3.md]: @sebgoa rephrased, and updated all other docs pages for AWS event sources. Could you please take another look?

File: docs/sources/awscodecommit.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ arn:aws:codecommit:{awsRegion}:{awsAccountId}:{repositoryName}
 
 ### API Credentials
 
-The TriggerMesh AWS CodeCommit event source authenticates calls to the AWS CodeCommit API using an [Access
-Key][accesskey]. The page at this link contains instructions to create an access key when signed either as the root user
-or as an IAM user. Take note of the **Access Key ID** and **Secret Access Key**, they will be used to create an instance
-of the event source.
+The TriggerMesh event source for Amazon CodeCommit authenticates calls to the AWS API using AWS Access Keys. The page
+[Understanding and getting your AWS credentials][accesskey] contains instructions to create access keys when signed
+either as the root user or as an IAM user. Take note of the **Access Key ID** and **Secret Access Key**, they will be
```
Comment (sebgoa) [inline docs/sources/awscodecommit.md:51]: when signed-in ?

**Review by sebgoa — APPROVED** [review]
cursory check of the other aws sources pages

**Pull request merged**